.github/workflows/build.yml

```diff
@@ -34,9 +27,9 @@ jobs:
       uses: actions/checkout@v2
       with:
         repository: xbmc/xbmc
-        ref: master
+        ref: Nexus
         path: xbmc
-    - name: Checkout pvr.hts repo
+    - name: Checkout add-on repo
```

Comment:
Curious, why is this now add-on instead of pvr.hts?

Reply:
Because the script can be reused for the other addons and then there is one place less to adapt.

Reply:
👍